### ISSUE TITLE
generated output contains only sixteenth notes

### ISSUE THREAD

**Issue by joaeba** [Ghadjeres/DeepBach#18]
Hi,
Thank you for sharing your DeepBach code.

I managed to generate some output with your model using the standard settings: 
`python3 deepBach.py -l 100`

However, the output contains only notes or pauses of duration 'sixteenth'. This pattern persists when I increase the number of iterations to 100 000. 

Many of the consecutive notes actually form sequences of the same pitch. Therefore might there be a problem in translating the hold symbol '__' back to note duration?

I actually had to install an older version of tensorflow to get keras 1.2.0 working with a tensorflow backend.  I downloaded version 0.11.0. Can you share the version you are using?

**Comment by Ghadjeres** [comment]
It seems that the model does not correspond to the dataset it has been trained on.
Have you downloaded the pretrained models together with the dataset?

**Comment by joaeba** [comment]
You're right, that was the problem: I built the bach dataset myself using deepBach.py and subsequently downloaded the pre-trained model without copying the associated dataset. Probably the dictionaries didn't match, which makes complete sense now that I think about it.

Thank you.
